cfpb/cfgov-refresh
# Update adult financial education urls

Adult fin-ed has been updated and released as Wagtail pages. This change
requires removing the old template url and redirecting the old url to
the new one. It also requires updating the navigation vars to avoid
unnecessary redirects.

## Changes

- Updated the urls.py to redirect the old url instead of pointing to the template.
- Updated the mega menu vars to use the new url

## Testing

1. navigate to `http://localhost:8000/adult-financial-education` and it should redirect.
2. Use the main nav to navigate to "Practitioner Resources" and then "Financial Education for Adults" 

## Checklist
